routes/faithRoutes.js

```diff
@@ -25,11 +35,26 @@ expressRouter.route('/actors')
       res.json({error: `${errorMsg} GET`});
     }
   })
-  
-  .put(async (req, res) => {
+
+  .put(async (req, res) => { // UPDATE ACTOR NAME in the possibility that actor changed their name
     try {
-      console.log(`${defaultMsg} PUT`);
-      res.json({message: `${defaultMsg} PUT`});
+      const actors = await db.sequelizeDB.query(getTable('actors'), {
+        type: sequelize.QueryTypes.SELECT
+      });
+      const actorName = `SELECT * FROM actors WHERE actor_name = "${req.body.actor_name}"`;
```

Comment:
I think you need a second argument for this put

maybe have this method take two args of actor_name and new_actor_name